MAUI/Maps/vector-layers/polygon-layer.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: Adding polygons in Maps control | Syncfusion
+description: Learn here all about adding the Polygons feature of Syncfusion .NET MAUI Maps (SfMaps) control to customize their appearances and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Polygons in .NET MAUI Maps (SfMaps)
+
+Polygon layer is a sublayer that renders a group of `MapPolygon` on `MapShapeLayer`. This section helps to learn about how to add the polygons and customize them.
+
+## Adding polygons
+
+The `Polygons` is a collection of `MapPolygon`. Every single `MapPolygon` connects multiple coordinates through a `Points` property.
```

Comment:
The `Polygons` is a collection of `MapPolygon.` Every single `MapPolygon` connects multiple coordinates through a `Points` property.

Reply:
Mentioned content is same as a UG default content.